webpack.config.js

```diff
@@ -36,6 +39,10 @@ module.exports = {
         exclude: /node_modules/,
         use: ["style-loader", "css-loader?modules=true", "sass-loader"],
       },
+      {
```

Comment:
Can you add some comments to explain the purpose of the 3 separate CSS loaders?

Reply:
Updated PR!